src/_scss/components/_roundedtoggle.scss

```diff
@@ -0,0 +1,71 @@
+.rounded-toggle__wrapper {
+  justify-content: baseline;
+  display:inline-flex;
+
+  .rounded-toggle__label {
+      color: $color-base;
+      font-weight: $font-semibold;
+      font-size: 14px;
```

Comment:
We usually use rem for units. But it's especially important with font-size for accessibility.  In this project  10 px = 1 rem.  A lot of code uses a bourbon library helper to convert px to rem so you can just pass the px value and the helper will convert to rem 

for this case the code would be :
font-size: rem(14);

usaspending-website/src/_scss/lib/bourbon/functions/_px-to-rem.scss